serverlessworkflow/modules/ROOT/pages/cloud/quarkus/build-workflow-images-with-tekton.adoc

```diff
@@ -1 +1,167 @@
-//= Building Workflow Images with Tekton Pipelines
\ No newline at end of file
+= Building Workflows Images with Tekton
+:compat-mode!:
+// Metadata:
+:description: Building Workflow Images with Tekton
+:keywords: kogito, workflow, serverless, kubernetes, minikube, openshift, containers, tekton, pipeline, ci
+
+This document describes how to build a Workflow Application on a Kubernetes cluster using the Tekton pipeline.
+
+
+.Prerequisites
+You have a git repository containing your workflow and the Containerfile to build the workflow as a container image
+and a namespace where you can install Tekton i.e. `custom-namespace`.
+
+.OptionalPrerequisites
+Tekton CLI:
+
+- On Openshift in the Operator pipelines is available a link to download the correct cli from the operaotr
+- On Minikube/kind/Kubernetes use this link https://tekton.dev/docs/cli/
+
+
+== Install Tekton pipeline
+
+If you are using Openshift, you can install the Red Hat Pipeline Operator.
```

Comment:
```suggestion
If you are using OpenShift, you can install the Red Hat Pipeline Operator.
```

Reply:
You **can** follow the easy path with the RH Operator or the hard path configuring manually the installation of Tekton